src/fcc_bridge_node_ros.cpp

```diff
@@ -0,0 +1,506 @@
+//
+// Created by Johan <job8197@thi.de> on 30.04.2024.
+//
+
+#include "fcc_bridge_node.hpp"
+
+// Libc header
+#include <chrono>
+#include <cinttypes>
+
+// CommonLib header
+#include "common_package/topic_names.hpp"
+
+namespace fcc_bridge {
+
+namespace {
+
+// Timer periods
+constexpr std::chrono::milliseconds FCC_TELEMETRY_PERIOD_5HZ{

```

Comment:
Having a constant for this makes sense, though naming your constant after the value doesn't

Reply:
I just wanted to pull the constant out of the code to make it visible that it is actually 5Hz.
I cannot think of a better name. 5Hz is the defining property of this timer.
Low Frequency timer is also not good because there might be a timer with even lower frequency in the future.